src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/Features/Ilr Changes/learner_changes_contract_type.feature

```diff
@@ -185,4 +185,75 @@ Scenario: DPP_965_05 - Non-Levy apprentice, provider edits contract type (ACT) i
+    Given levy balance > agreed price for all months
+	And the apprenticeship funding band maximum is 9000
+
+	And the following commitments exist:
+		| commitment Id | version Id | ULN       | start date | end date   | framework code | programme type | pathway code | agreed price | status | effective from | effective to |
+		| 1             | 1          | learner a | 01/08/2017 | 01/08/2018 | 403            | 2              | 1            | 9000         | Active | 01/08/2017     |              |
+        
+	When an ILR file is submitted for period R08 with the following data:
+        | ULN       | learner type       | agreed price | start date | planned end date | actual end date | completion status | aim type         | aim sequence number | aim rate | framework code | programme type | pathway code | contract type | contract type date from |
+        | learner a | programme only DAS | 9000         | 06/08/2017 | 20/08/2018       |                 | continuing        | programme        | 2                   |          | 403            | 2              | 1            | DAS           | 06/08/2017              |
+        | learner a | programme only DAS |              | 06/08/2017 | 20/08/2018       |                 | continuing        | maths or english | 1                   | 471      | 403            | 2              | 1            |               |                         |
+        
+    And an ILR file is submitted for period R09 with the following data:
+        | ULN       | learner type           | agreed price | start date | planned end date | actual end date | completion status | aim type         | aim sequence number | aim rate | framework code | programme type | pathway code | contract type | contract type date from |
+        | learner a | programme only Non-DAS |              | 06/08/2017 | 20/08/2018       |                 | continuing        | maths or english | 1                   | 471      | 403            | 2              | 1            | Non-DAS       | 06/08/2017              |
+        | learner a | programme only Non-DAS | 9000         | 06/08/2017 | 20/08/2018       |                 | continuing        | programme        | 2                   |          | 403            | 2              | 1            |               |                         |
+		
+	Then the provider earnings and payments break down as follows:
+        | Type                                    | 08/17  | 09/17  | 10/17  | 11/17  | 12/17  | 01/18  | 02/18  | 03/18  | 04/18   | 05/18  |
+        | Provider Earned Total                   | 639.25 | 639.25 | 639.25 | 639.25 | 639.25 | 639.25 | 639.25 | 639.25 | 639.25  | 639.25 |
+        | Provider Earned from SFA                | 0      | 639.25 | 639.25 | 639.25 | 639.25 | 639.25 | 639.25 | 639.25 | 639.25  | 639.25 |
```

Comment:
Provider earned from SFA should not be offset. Althrough no assertion is done on this line